web/styles/modal.css

```diff
@@ -554,3 +554,17 @@
         }
     }
 }
+
+#onboarding-video-modal {
+    width: 70%;
+
+    #onboarding-video-wrapper {
+        display: flex;
+        justify-content: center;
+    }
+
+    video {
+        max-width: 100%;
+        cursor: pointer;
```

Comment:
This is an addition since yesterday, because Alya mentioned we don't plan to autoplay.

```diff
+cursor: pointer
```